element_lab/export/nwb.py

```diff
@@ -1,5 +1,6 @@
 from element_lab import lab
 
+raise NotImplementedError('The updated Element Lab "project" schema is not ready for NWB export, stay tuned!')
```

Comment:
If we are keeping the projects tables in the `lab` schema for backwards compatibility, then would these functions still work?

Reply:
Yes, it would still work with the previous design, without the project schema. I'm not sure why @ttngu207 opted for `NotImplementedError` here. I can embed this under a conditional that checks if the project schema is active, or revise to be a warning.